volumegroup/README.md

```diff
@@ -0,0 +1,455 @@
+# Container Storage Interface (CSI)
```

Comment:
This is not CSI directly, call it CSI-Addons instead?

As this is heavlly based on CSI and Kubernetes proposals for VolumeGroups, it would be nice to mention that and maybe provide links to the abandoned/rejected PRs/issues.

Reply:
done